metals/src/main/scala/scala/meta/internal/decorations/SemanticdbTreePrinter.scala

```diff
@@ -86,6 +86,8 @@ class SemanticdbTreePrinter(
         } else {
           s"${printType(tpe)} {${printScope(scope)}}"
         }
+      case s.MatchType(scrutinee, _) =>
+        s"${printType(scrutinee)} match { ... }"
```

Comment:
Maybe
```suggestion
      case s.MatchType(scrutinee, cases) =>
        s"${printType(scrutinee)} match { ${cases.size} cases }"
```
or even some heuristic like if cases.size > 2 then print only size, otherwise print all cases (2 of them)?